src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+    private int maxNumber = 100;
```

Comment:
make it constant - static final + MAX_NUMBER, and change to 101, because the value passed to randomizer is excluded itself